README.md

```diff
@@ -95,12 +110,14 @@ See https://techcommunity.microsoft.com/t5/hardware-dev-center/nx-exception-for-
 [your text here]
 
 *******************************************************************************
-### If shim is loading GRUB2 bootloader what exact implementation of Secureboot in GRUB2 do you have? (Either Upstream GRUB2 shim_lock verifier or Downstream RHEL/Fedora/Debian/Canonical-like implementation)
+### What exact implementation of Secure Boot in GRUB2 do you have? (Either Upstream GRUB2 shim_lock verifier or Downstream RHEL/Fedora/Debian/Canonical-like implementation)
+Skip this, if you're using a different second-stage bootloader.
```

Comment:
Say "not using GRUB2" if you are using a different second-stage bootloader.

maybe? I'd rather be 100% clear :-)